pkg/arvo/app/invite-store.hoon

```diff
@@ -1,184 +1,172 @@
 ::  invite-store [landscape]
-/+  *invite-json, default-agent, dbug
+/-  *invite-store
+/+  default-agent, dbug
 |%
 +$  card  card:agent:gall
-::
 +$  versioned-state
-  $%  state-zero
+  $%  state-0
   ==
 ::
-+$  state-zero
-  $:  %0
-      =invites
-  ==
++$  state-0  [%0 =invites]
 --
 ::
-=|  state-zero
+=|  state-0
 =*  state  -
 %-  agent:dbug
 ^-  agent:gall
-=<
-  |_  bol=bowl:gall
-  +*  this      .
-      inv-core  +>
-      ic        ~(. inv-core bol)
-      def       ~(. (default-agent this %|) bol)
-  ++  on-init   on-init:def
-  ++  on-save   !>(state)
-  ++  on-load
-    |=  old=vase
-    `this(state !<(state-zero old))
+::
+|_  =bowl:gall
++*  this      .
+    def       ~(. (default-agent this %|) bowl)
+++  on-init   on-init:def
+++  on-save   !>(state)
+++  on-load
+  |=  old=vase
+  `this(state !<(state-0 old))
+::
+++  on-agent  on-agent:def
+++  on-arvo   on-arvo:def
+++  on-fail   on-fail:def
+::
+++  on-watch
+  |=  =path
+  ^-  (quip card _this)
+  =/  cards=(list card)
+    ?+    path  (on-watch:def path)
+        [%all ~]      [%give %fact ~ %invite-update !>([%initial invites])]~
+        [%updates ~]  ~
+        [%invitatory *]
+      =/  inv=invitatory  (~(got by invites) t.path)
+      [%give %fact ~ %invite-update !>([%invitatory inv])]~
+    ==
+  [cards this]
+::
+++  on-poke
+  |=  [=mark =vase]
+  ^-  (quip card _this)
+  |^
+  ?>  (team:title our.bowl src.bowl)
+  =^  cards  state
+    ?+  mark  (on-poke:def mark vase)
+      %invite-action  (poke-invite-action !<(invite-action vase))
+    ==
+  [cards this]
   ::
-  ++  on-poke
-    |=  [=mark =vase]
-    ^-  (quip card _this)
-    ?>  (team:title our.bol src.bol)
-    =^  cards  state
-      ?+  mark  (on-poke:def mark vase)
-        %json           (poke-invite-action:ic (json-to-action !<(json vase)))
-        %invite-action  (poke-invite-action:ic !<(invite-action vase))
-      ==
-    [cards this]
+  ++  poke-invite-action
+    |=  action=invite-action
+    ^-  (quip card _state)
+    ?-  -.action
+        %create   (handle-create action)
+        %delete   (handle-delete action)
+        %invite   (handle-invite action)
+        %accept   (handle-accept action)
+        %decline  (handle-decline action)
+    ==
   ::
-  ++  on-watch
-    |=  =path
-    ^-  (quip card _this)
-    =/  cards=(list card)
-      ?+    path  (on-watch:def path)
-          [%all ~]      [%give %fact ~ %invite-update !>([%initial invites])]~
-          [%updates ~]  ~
-          [%invitatory *]
-        =/  inv=invitatory  (~(got by invites) t.path)
-        [%give %fact ~ %invite-update !>([%invitatory inv])]~
-      ==
-    [cards this]
+  ++  handle-create
+    |=  act=invite-action
+    ^-  (quip card _state)
+    ?>  ?=(%create -.act)
```

Comment:
I'd rather calling `+handle-create` with `(handle-create +.action)` and then you can dodge this `?>` as well as avoiding the mildly unnecessary `act` face. Again, up to you

Reply:
Good point. I straight copied the old methods, but that would be cleaner. I'll do that before we merge this.